Translations/it.po

```diff
@@ -11505,61 +11500,120 @@ msgstr "Celia"
 msgstr "Per le catacombe"
 
-#: Source/trigs.cpp:363
+#: Source/trigs.cpp:364
 msgid "Down to caves"
 msgstr "Giù alle caverne"
 
-#: Source/trigs.cpp:373
+#: Source/trigs.cpp:374
 msgid "Down to hell"
 msgstr "Giù all'inferno"
 
-#: Source/trigs.cpp:385
+#: Source/trigs.cpp:386
 msgid "Down to Hive"
 msgstr "Giù all'Alveare"
 
-#: Source/trigs.cpp:397
+#: Source/trigs.cpp:398
 msgid "Down to Crypt"
 msgstr "Giù nella Cripta"
 
-#: Source/trigs.cpp:413 Source/trigs.cpp:493 Source/trigs.cpp:540
-#: Source/trigs.cpp:635
+#: Source/trigs.cpp:414 Source/trigs.cpp:494 Source/trigs.cpp:541
+#: Source/trigs.cpp:636
 msgid "Up to level {:d}"
 msgstr "Su al livello {:d}"
 
-#: Source/trigs.cpp:415 Source/trigs.cpp:470 Source/trigs.cpp:522
-#: Source/trigs.cpp:601 Source/trigs.cpp:618 Source/trigs.cpp:665
+#: Source/trigs.cpp:416 Source/trigs.cpp:471 Source/trigs.cpp:523
+#: Source/trigs.cpp:602 Source/trigs.cpp:619 Source/trigs.cpp:666
 msgid "Up to town"
 msgstr "Su in città"
 
-#: Source/trigs.cpp:426 Source/trigs.cpp:504 Source/trigs.cpp:557
-#: Source/trigs.cpp:582 Source/trigs.cpp:647
+#: Source/trigs.cpp:427 Source/trigs.cpp:505 Source/trigs.cpp:558
+#: Source/trigs.cpp:583 Source/trigs.cpp:648
 msgid "Down to level {:d}"
 msgstr "Giù al livello {:d}"
 
-#: Source/trigs.cpp:438
+#: Source/trigs.cpp:439
 msgid "Up to Crypt level {:d}"
 msgstr "Giù al livello della Cripta {:d}"
 
-#: Source/trigs.cpp:453
+#: Source/trigs.cpp:454
 msgid "Down to Crypt level {:d}"
 msgstr "Giù al livello della Cripta {:d}"
 
-#: Source/trigs.cpp:569
+#: Source/trigs.cpp:570
 msgid "Up to Nest level {:d}"
 msgstr "Su al livello del Covo {:d}"
 
-#: Source/trigs.cpp:678
+#: Source/trigs.cpp:679
 msgid "Down to Diablo"
 msgstr "Giù da Diablo"
 
-#: Source/trigs.cpp:711 Source/trigs.cpp:725 Source/trigs.cpp:739
+#: Source/trigs.cpp:712 Source/trigs.cpp:726 Source/trigs.cpp:740
 msgid "Back to Level {:d}"
 msgstr "Torna al Livello {:d}"
+
+#~ msgid "Menu"
+#~ msgstr "Menu"
+
+#~ msgid "Inv"
+#~ msgstr "Inv"
+
+#~ msgid "Map"
+#~ msgstr "Mappa"
+
+#~ msgid "Char"
+#~ msgstr "Profilo"
+
+#~ msgid "Spells"
+#~ msgstr "Magie"
+
+#~ msgid "Quests"
+#~ msgstr "Missioni"
+
+#~ msgid "Resist Fire: 75% MAX"
+#~ msgstr "Resistenza Fuoco: 75% MAX"
+
+#~ msgid "Resist Lightning: 75% MAX"
+#~ msgstr "Resistenza Fulmine: 75% MAX"
+
+#~ msgid "Resist Magic: 75% MAX"
+#~ msgstr "Resistenza Magia: 75% MAX"
+
+#~ msgid "Resist All: 75% MAX"
+#~ msgstr "Resistenza Tutto: 75% MAX"
+
+#~ msgid "dam: {:d}  Dur: {:d}/{:d}"
+#~ msgstr "dan: {:d}  Dur: {:d}/{:d}"
+
+#~ msgid "dam: {:d}-{:d}  Dur: {:d}/{:d}"
+#~ msgstr "dan: {:d}-{:d}  Dur: {:d}/{:d}"
+
+#~ msgid "Magic "
+#~ msgstr "Magia "
+
+#~ msgid "Lightning "
+#~ msgstr "Fulmine "
+
+#~ msgid "MAX"
+#~ msgstr "MAX"
+
+#, c-format
+#~ msgid ",%03d"
+#~ msgstr ",%03d"
+
+#, c-format
+#~ msgid "%i gold"
+#~ msgstr "%i monete"
+
+#~ msgid "Experience: "
+#~ msgstr "Esperienza: "
+
+#~ msgid "Next Level: "
+#~ msgstr "Livello Successivo: "
```

Comment:
This was not intentionally removed, I have fixed it in master now. If possible could you update the translation to still include these lines :)